index.html

```diff
@@ -63,9 +63,9 @@
             <li>
               <!-- <img src="./assets/images/keiko-logo.png" class="logo" alt="Keiko Corp" /> -->
             </li>
-            <li class="active" id=home> <a href="/">
+            <li class="active" id=home> <a href="./index.html">
```

Comment:
Hello @Soumi150 , well this will fix the issue, but it will use redirect(refresh) every time, 
instead **we can use** :-
`<li class="active" id=home> <a href="#">`

1. This will **save time** and just drag on top of page without redirect (refresh)
2. I have checked, **it works** even if we are on a differect page (About or Contact)

 
_(Different issue :- Blog and Job listings are currently not correct)_